### PR TITLE
Fix WebAuthn Step2

### PR DESCRIPTION
**Pull request by dzatoah**
TODO:
- [x] `piTokenOrigin` is available on our customer branch, but not here. Should we add it here too and remove it on customer-branch?

### CODE REVIEW

**Comment by sunweaver** [comment]
Not sure what you mean, but let me answer like this: piTokenOrigin must be part of the generic gosa-plugins-privacyidea implementation.

If gosa-core receives updates for MFA from GONICUS for the login procedure, they will also have to use piTokenOrigin and send it in the header to the PI server.

File: personal/privacyidea/class_PrivacyIdeaUtils.inc [hunk 1]
```diff
@@ -600,8 +600,8 @@ class PrivacyIdeaUtils implements PILog
     {
         assert(is_string($key));
 
-        // Get config value from LDAP-Tree, gosa config file (gosa.conf) or
-        // defaults from class_schoolmgr.inc
+        // Get config value from LDAP tree, gosa config file (gosa.conf) or
+        // defaults from class_mfaAccount.inc
```
Comment (sunweaver) [inline personal/privacyidea/class_PrivacyIdeaUtils.inc:604]: ouch! copy+waste. Nice catch.

**Review by sunweaver — DISMISSED** [review]
otherwise LGTM

**Comment by dzatoah** [comment]
> otherwise LGTM

Please review now. Thanks :)

File: personal/privacyidea/class_mfaAccount.inc [hunk 2]
```diff
@@ -974,6 +974,16 @@ class mfaAccount extends plugin
                 "mandatory"   => true,
                 "description" => _("privacyIDEA server URL.")
             ),
+            array(
+                "name"        => "piTokenOrigin",
+                "type"        => "string",
+                "check"       => "gosaProperty::isString",
+                "group"       => "core",
+                "default"     => "http://tjener.intern/",
```
Comment (sunweaver) [inline personal/privacyidea/class_mfaAccount.inc:982]: GOsa² != Debian Edu. Rather some generic thing: ```gosa.mysite.tld```

Comment (sunweaver) [reply personal/privacyidea/class_mfaAccount.inc:982]: This always requires adjustment, I guess.

**Comment by dzatoah** [comment]
R 4 r.

File: personal/privacyidea/class_mfaAccount.inc (hunk 2 above)
Comment (gber) [reply personal/privacyidea/class_mfaAccount.inc]: Please do not use `mysite.tld` but one of the reserved names `example.(com|net|org)` which exists for this purpose.

**Review by sunweaver — APPROVED** [review]
LGTM now. Thanks + Sigh!

**Pull request merged**